fast-af/fast-checkout-woocommerce
# PINT-1010 - Add fast as payment gateway

# Description

Some 3rd-party integrations require Fast to be listed as a payment gateway in WooCommerce. This update adds a Fast payment gateway object in WooCommerce.

# Testing instructions

1. Login to staging.
2. Verify that Fast is included as a payment gateway in the WooCommerce settings.
3. When the Fast payment gateway is enabled, verify that it shows up as a payment option in checkout.

# Checks
